collect_app/src/main/java/org/odk/collect/android/preferences/AdminPreferencesActivity.java

```diff
@@ -19,11 +19,11 @@
 import android.os.Bundle;
 import android.preference.PreferenceActivity;
 import android.preference.PreferenceManager;
-import android.util.Log;
+//import android.util.Log;
```

Comment:
You don't need to comment out code that's no longer needed, you can just go ahead and remove it! There will be a record of how it used to be in version control. Please remove the two unused imports.